tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb

```diff
@@ -1023,11 +923,11 @@
    "execution_count": null,
    "id": "96542ae9",
    "metadata": {
-    "execution": {}
+    "scrolled": true
    },
    "outputs": [],
    "source": [
-    "selected_words_2 = [..., ..., ..., ..., ...]\n",
```

Comment:
I think this needs to stay as is else it is not possible to iterate over `selected_words_2` in the next line